### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by SrUca** [Jboymercs/nether_backport#7]
So... i think it'd be neat to get rid of the Pigmen Zombi and just make them be the Zombified Piglins since rn i don't get why there should be this two mobs, since one if the upgrade of the other.

Another thing it'd like to suggest is to make dead animations since rn when you kill one it just stand still and then dissapear.

**Comment by darkheadmaster132** [comment]
a lot of mods are depended on the zombie pigmen tho

**Comment by SrUca** [comment]
> a lot of mods are depended on the zombie pigmen tho

Yeah but if the mob is let intact but just reskined and renamed i don't think there would be any problem

**Comment by Jboymercs** [comment]
I feel like the decision over this due to Zombie Pigmen still playing a part in other mods will not have this request be taken. Thank you for suggesting though!
